inference-engine/src/mkldnn_plugin/nodes/mkldnn_non_max_suppression_node.cpp

```diff
@@ -111,8 +111,8 @@ void MKLDNNNonMaxSuppressionNode::initSupportedPrimitiveDescriptors() {
         check1DInput(inputShape_SOFTNMSSIGMA, supportedFloatPrecision, "soft_nms_sigma", NMS_SOFTNMSSIGMA);
     }
 
-    checkOutput(outputShape_SELECTEDINDICES, supportedIntOutputPrecision, "selected_indices", NMS_SELECTEDINDICES);
-    checkOutput(outputShape_SELECTEDSCORES, supportedFloatPrecision, "selected_scores", NMS_SELECTEDSCORES);
+    checkOutput(outputShape_SELECTEDINDICES.getDims(), supportedIntOutputPrecision, "selected_indices", NMS_SELECTEDINDICES);
```

Comment:
maybe better to rewrite ```check*``` methods with ```Shape``` instead ```SizeVector```?

Reply:
Fixed. Thank you!